doc/source/serve/configure-serve-deployment.md

```diff
@@ -62,79 +58,42 @@ applications:
     ray_actor_options:
       num_cpus: 0.2
       num_gpus: 0.0
-
 ```
 
-## Overriding deployment settings
+### Overriding deployment settings
 
-The order of priority is (from highest to lowest):
+For each individual parameter, the order of priority is (from highest to lowest):
 
 1. Serve Config file
-2. `.options()` call in python code referenced above
-3. `@serve.deployment` decorator in python code
-4. Serve defaults
+2. Application code (either through the `@serve.deployment` decorator or through `.options()`)
+3. Serve defaults
 
-For example, if a deployment's `num_replicas` is specified in the config file and their graph code, Serve will use the config file's value. If it's only specified in the code, Serve will use the code value. If the user doesn't specify it anywhere, Serve will use a default (which is `num_replicas=1`).
+In other words, if a parameter for a deployment is specified in the config file and the application code, Serve will use the config file's value. If it's only specified in the code, Serve will use the value specified in the code. If the parameter is specified anywhere, Serve will use the default for that parameter.
 
-Keep in mind that this override order is applied separately to each individual parameter.
-For example, if a user has a deployment `ExampleDeployment` with the following decorator:
+For example, if you have the following application code which contains a single deployment `ExampleDeployment`:
 
 ```python
-@serve.deployment(
-    num_replicas=2,
-    max_concurrent_queries=15,
-)
+@serve.deployment(num_replicas=2, graceful_shutdown_timeout_s=6)
 class ExampleDeployment:
     ...
+
+example_app = ExampleDeployment.bind()
 ```
 
-and the following config file:
+Then suppose you deploy your application with the following config file:
 
 ```yaml
-...
-
-deployments:
-
-    - name: ExampleDeployment
-      num_replicas: 5
-
-...
+applications:
+  - name: default
+    import_path: models:example_app 
+    deployments:
+      - name: ExampleDeployment
+        num_replicas: 5
 ```
 
-Serve sets `num_replicas=5`, using the config file value, and `max_concurrent_queries=15`, using the code value (because `max_concurrent_queries` wasn't specified in the config file). All other deployment settings use Serve defaults because the user didn't specify them in the code or the config.
+Serve uses `num_replicas=5` from the value set in the config file and `graceful_shutdown_timeout_s=6` from the value set in the application code. All other deployment settings use Serve defaults because the user didn't specify them in the code or the config. For instance, `health_check_period_s=10` since by default Serve health checks deployments once every 10 seconds.
```

Comment:
```suggestion
Serve uses `num_replicas=5` from the value set in the config file and `graceful_shutdown_timeout_s=6` from the value set in the application code. All other deployment settings use Serve defaults because they're not specified in the code or the config. For instance, `health_check_period_s=10` since by default Serve health checks deployments once every 10 seconds.
```